### ISSUE TITLE
bootstrap.min.css is saying i changed it and i cannot edit or delete the file 

### ISSUE THREAD

**Issue by SgtBatten** [causefx/Organizr#38]
I am on windows 10. I am the only user account on the PC. I am the administrator. No matter what ive tried (changing permissions, taking ownership, ensuring i am the admin) neither git nor I can remove or edit this file. I notice that it recieved an auto update yesterday. perhaps thats the cause. no other file is affected though. Git Gui cant even display the supposed changed between my file and yours. it says access denied.

May be related or may not but it coincided with the error:
I changed all tabs to not have guest permissions. Since then 2 mobile devices and 1 laptop are malfunctioning. I can login or logout but after loggin in i can only see the do you want to log out prompt but cant press no, only yes. After loggin out I can see the login dialog but not access any of the guest tabs.
[Pic of the login page](http://imgur.com/vfERgBQ)
You will also see that it is offset, the top bar icons are all on the left instead of some on the right. some weird white line is in the right and half a word is also displayed.

I have used one of two working decktops to change it back but it hasnt helped.

**Comment by causefx** [comment]
You can edit any other file except this one?

**Comment by SgtBatten** [comment]
Yep I can edit anything else.

Version of what sorry?

Permissions wise everyone has everything set to allow except for special permissions. 

As for Bootstrap.min.css I don't even have permission to view who has permissions. Access is denied. I even tried running CMD as admin and using icacls command but it says access is denied.

**Comment by causefx** [comment]
The version of Organizr.  Since you can't login, open up settings.php its on line 1639.  This makes no sense that you cannot edit as Admin.  Has anything changed on your nginx settings, i doubt that has anything to do with it as you cant access the file on windows.

**Comment by causefx** [comment]
you could also try a tool like this:

http://www.majorgeeks.com/mg/getmirror/iobit_unlocker,1.html

**Comment by SgtBatten** [comment]
We'll interestingly you have solved it again.

I tried something different and renamed the CSS folder but it said something inside was in use by bother program. That's was new. Everything else before has simply said access denied.

So based on your suggestion I stopped nginx. The bootstrap.min.CSS just vanished. Self deleted itself when nginx stopped.

So I just resynced with git and started nginx. Good as gold.
